modules/lib/generators.nix

```diff
@@ -1,6 +1,49 @@
 { lib }:
 
-{
+rec {
+  toHyprconf = with lib;
```

Comment:
Imo lib functions should be inherited, as it's being done in the `toKDL` function, knowing `with x;` is an anti-pattern in the Nix language.